android/assets/jsons/translations/template.properties

```diff
@@ -1760,6 +1760,7 @@ Move =
 After an unknown civilization entered the [eraName], we have recruited [spyName] as a spy! = 
 We have recruited [spyName] as a spy! = 
 Your spy [spyName] has leveled up! = 
+Your spy [spyName] has leveled up [amount] times! = 
```

Comment:
This should also be time(s)

Reply:
In this case I am using the previous text `Your spy [spyName] has leveled up!` when we level up only one time. It looks a bit nicer and will be used much more often.